my-dsl.md

```diff
@@ -1,38 +1,105 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
+_What is the name of the language? Link the name to its webpage
 (if appropriate)._
 
+[Dockerfile Command Language](https://docs.docker.com/engine/reference/builder/)
 
 # Domain
 _Describe the language's domain in five words._
 
+Efficient portable software runtime environments.
 
 # Computational model
-_We don't yet have a great definition of the term "computational model". 
-For now, try to come up with the clearest, most concise explanation of 
+_We don't yet have a great definition of the term "computational model".
+For now, try to come up with the clearest, most concise explanation of
 what happens when a program in your DSL runs._
 
+A user describes the overall build of a container&mdash;a sort of lightweight
+virtual appliance&mdash;in a text file called a Dockerfile with limited text
+commands. When a user builds an image from the Dockerfile, the text commands are
+parsed and with each step a partial container is built representing the state of
+the machine&mdash;the Docker commands are converted to lower-level system
+commands as needed. After the final command, the user can use the built machine
+just as they would a virtual appliance without worrying about the intermediate
+steps. If the Dockerfile is edited, the image builder will find the modified
+line, and only rebuild and commit images that came from commands with and after
+that line.
```

Comment:
I'm not familiar with Docker (on the todo list...), but this sounds like a glorified config file with some Makefile thrown in. As such, it seems like it would be just barely a DSL - would you agree, or am I missing something?